storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1014,6 +1029,10 @@ void Archived_Redo_Log_Monitor::thread_func() {
 
   if (res == nullptr) {
     xb::info() << "Redo Log Archiving is not used.";
+    if (!innodb_redo_log_archive) {
```

Comment:
Since we are repeating the same code in multiple places, we might create a member function to execute the cleanup.

Reply:
I will create it